tensorflow/python/ops/math_ops.py

```diff
@@ -3212,6 +3212,12 @@ def sigmoid(x, name=None):
 
   Returns:
     A Tensor with the same type as `x`.
+  
+  Usage Example:
+    >>> x = tf.constant([125.92, -76.32, 44.12, -87.3, -82.72, 0.0, 112.23], dtype=tf.float32)
```

Comment:
Insert an empty line between `Usage Example:` and the doctest. Also, don't indent the doctest.

This way, it will also show up nicely on the documentation website.

Reply:
Also, I think there are too many values, let's try with only `-128, 0, 128`